portfolio/src/main/webapp/index.html

```diff
@@ -63,5 +63,11 @@ <h1>Michaela Murray's Portfolio</h1>
     <div id="inner-content" style="display:none;">
       <div id="my-info"></div>
     </div>
+    <form action="/data" method="POST">
+      <p>Submit a comment!</p>
+      <textarea name="text-input"></textarea>
+      <br/><br/>
+      <input type="submit"/>
```

Comment:
I believe `input` is considered a void element so there is no need to close it. @ccondit to confirm.
```suggestion
      <input type="submit">
```